udata/forms/fields.py

```diff
@@ -180,6 +180,19 @@ def __init__(self, *args, **kwargs):
         self.stacked = kwargs.pop('stacked', False)
         super(BooleanField, self).__init__(*args, **kwargs)
 
+    def process_formdata(self, valuelist):
+        if not valuelist:
+            return 
+
+        if valuelist[0] is None:
```

Comment:
Do we need to deal with `None`? Seems like this code isn't used if value is None, right?